doc/source/netscript/basicfunctions/getRunningScript.rst

```diff
@@ -0,0 +1,88 @@
+getRunningScript() Netscript Function
+=====================================
```

Comment:
Have you tried building docs locally? Not sure if it will complain about line length here

Reply:
It does not. :)